Sources/StytchCore/Generated/StytchClient.UserManagement.deleteFactor+AsyncVariants.generated.swift

```diff
@@ -0,0 +1,33 @@
+// Generated using Sourcery 1.8.0 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+import Combine
+import Foundation
+
+public extension StytchClient.UserManagement {
+    /// Deletes, by id, an existing authentication factor associated with the current user.
+    func deleteFactor(_ factor: AuthenticationFactor, completion: @escaping Completion<UserResponse>) {
```

Comment:
Android exposes explicit endpoints for each factor (of the ones we support). Would you like us to standardize like you have with one method and parameter types?

Reply:
@jhaven-stytch This difference feels somewhat minor to me, so it's not necessarily a problem to keep it as it is on your end. 

On my side, It feels like more idiomatic Swift by taking advantage of the type system and Swift's powerful `enum`s, so I think it makes sense for the Swift SDK, but again I don't think it's critical to be 100% aligned, since we're ...call it 95% aligned here. It also allows me to reduce code bloat by generating less code for async variants since it's a single method vs a method per factor.

If this feels like something that you could do similarly in the Kotlin world, then I'd say let's go ahead and align the native SDKs. If you want to charge down that path, it's also worth nothing that I've done something similar to the OTP send and loginOrCreate methods via a `DeliveryMethod` enum.

Reply:
Yeah, the codebloat is palpable on the Android side. That plus being even closer in alignment makes me want to update Android. Fortunately the Android UserManagement stuff hasn't been released yet, so nows the time to do it. Ticket incoming!

Reply:
sounds great!